src/firefly/js/ui/CheckboxGroupInputField.jsx

```diff
@@ -18,32 +18,33 @@ function convertValue(value,options) {
 
 export function CheckboxGroupInputFieldView({fieldKey, onChange, label, tooltip:toggleBoxTip, slotProps, type,
                                              options, alignment:orientation, value:fieldValue, sx}) {
-
     return (
         <Tooltip title={toggleBoxTip} sx={sx} {...slotProps?.tooltip}>
             <FormControl orientation={orientation}  {...slotProps?.control}>
-                {label && <FormLabel {...slotProps?.label}>{label}</FormLabel>}
-                {/*following should be nested in a <FormGroup/> once joy-ui exposes it: https://mui.com/material-ui/react-checkbox/#formgroup*/}
-                <Stack className='ff-Checkbox-container' spacing={orientation==='vertical'?1:2} direction={orientation==='vertical' ? 'column' : 'row'}>
-                    {options.map( ({value,label,tooltip}) => {
-                        const cb= type==='switch' ?
-                            (<Switch {...slotProps?.input}
-                                       {...{ size:'sm', name:fieldKey, key:value, value,
-                                           endDecorator: label,
-                                           checked:isChecked(value,fieldValue), onChange, label, }} />)
-                            :
-                            (<Checkbox {...slotProps?.input}
-                            {...{ className:'ff-Checkbox-item', size:'sm', name:fieldKey, key:value, value, checked:isChecked(value,fieldValue), onChange, label, }} />);
-                        return (
-                            <FormControl key={value}>{
-                                //until https://github.com/mui/material-ui/issues/37764 & related issues are fixed,
-                                //we need to wrap `cb` in a FormControl as a workaround: https://stackoverflow.com/a/66738444/8252556
-                                tooltip
-                                    ? <Tooltip {...{title: toggleBoxTip, key: value}}> {cb} </Tooltip>
-                                    : cb
-                            }</FormControl>
-                        );
-                    })}
+                <Stack direction='row' alignItems='center' spacing={2}>
```

Comment:
This is most likely not right.  It's a reusable component and cannot be styled like this.  I think direction is handled by FormControl orientation.

Reply:
I had totally missed this. Got rid of it now! Just setting `orientation` (alignment) directly from where `CheckboxGroupInputField` is called in FinderChart.